src/xml_interface.cpp

```diff
@@ -37,4 +35,28 @@ get_node_value(pugi::xml_node node, const char *name)
   return value;
 }
 
+std::vector<std::string>
+split(const std::string in)
```

Comment:
`const` is superfluous since `in` will be copied. Probably what you want is `const std::string& in`

Reply:
Also, this function probably should be in some other file since it's not XML-specific.